.eslintrc.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
 
         "no-duplicate-imports": "error",
         "no-console": "error",
-        "no-magic-numbers": "error",
+        "no-magic-numbers": ["error", { "ignoreArrayIndexes": true }],
```

Comment:
Added since I was getting a linting error when referencing the first position by index.